kaldb/pom.xml

```diff
@@ -14,17 +14,17 @@
         <uberjar.name>kaldb</uberjar.name>
         <http.client.version>4.5.13</http.client.version>
         <http.core.version>4.4.15</http.core.version>
-        <protobuf.version>3.20.1</protobuf.version>
-        <grpc.version>1.46.0</grpc.version>
-        <armeria.version>1.16.0</armeria.version>
-        <micrometer.version>1.8.5</micrometer.version>
-        <kafka.version>2.5.1</kafka.version>
+        <protobuf.version>3.21.1</protobuf.version>
+        <grpc.version>1.47.0</grpc.version>
+        <armeria.version>1.17.0</armeria.version>
+        <micrometer.version>1.9.1</micrometer.version>
+        <kafka.version>3.2.0</kafka.version>
```

Comment:
Let's leave Kafka version as is. Also, unless there is a known advantage let's not bump up the versions of the software that we don't need to use it.

Reply:
I had to upgrade this due to the fact we need to upgrade `kafka-junit`. We need to upgrade `kafka-junit` because of an issue with `jackson.databind` and Scala version mismatches (see the support matrix here - https://github.com/charithe/kafka-junit).

`jackson.databind` has to be updated due to multiple security vulnerabilities, and 2.12 is the oldest available that doesn't have those vulnerabilities.

The dependencies listed here were the minimum set I was able to find that satisfied upgrading Armeria and its associated dependencies (ie, micrometer, java grpc, brave), and addressed the vulnerable packages.